Translations/pt_BR.po

```diff
@@ -7305,11 +7300,11 @@ msgid ""
 "have little use for it. May it protect you against the dark powers below. Go "
 "with the Light, my friend..."
 msgstr ""
-"Você salvou a minha alma da danação, e por isso estou agora em dívida. Se "
-"houver alguma forma pela qual eu possa lhe recompensar do além-túmulo eu a "
-"encontrarei, mas, por agora, tome o meu elmo. Na jornada que estou prestes a "
-"seguir ele terá pouca serventia. Que lhe proteja contra os poderes sombrios "
-"abaixo. Vá com a Luz, meu(inha) amigo(a)..."
+"Salvou a minha alma da condenação, e por isso estou agora em dívida com "
```

Comment:
I got what you did here to avoid repeating but since it's a transcription, it should be fine.
```suggestion
"Você salvou a minha alma da danação, e por isso estou agora em dívida com "
```